packages/api/src/store/db.ts

```diff
@@ -41,6 +42,11 @@ export type DBSession = WithID<Session> &
 
 type Table<T> = BaseTable<WithID<T>>;
 
+type QueryHistogramLabels = {
+  query: string;
+  result?: string;
```

Comment:
This one doesn't even need to be optional anymore since we're initializing the labels with it ✨ 
```suggestion
  result: string;
```